en/02_Developer_Guides/10_Email/index.md

```diff
@@ -261,6 +261,25 @@ $email = Email::create($from, $to, $subject, $body);
 $email->replyTo('reply@example.com');
 ```
 
+## Catching email send failure exceptions
+
+If you wish to handle email send failures then you can wrap `$email->send()` with a try/catch block that catches the Symfony Mailer `TransportExceptionInterface`.
+
+```php
+use SilverStripe\Control\Email\Email;
+use Symfony\Component\Mailer\Exception\TransportExceptionInterface;
+
+$email = Email::create('from@example.com', 'to@example.com', 'My subject');
+$email->text('My plain text email content');
+try {
+    $email->send();
+} catch (TransportExceptionInterface $e) {
+    // handle exception
+}
+```
+
+For more information, refer to [handling sending failures](https://symfony.com/doc/current/mailer.html#handling-sending-failures) in the [symfony/mailer docs](https://symfony.com/doc/current/mailer.html).
```

Comment:
```suggestion
For more information, refer to [handling sending failures](https://symfony.com/doc/current/mailer.html#handling-sending-failures) in the symfony/mailer docs.
```
That link is already to the docs. Having two different links to two different parts of the doc for the same thing is just confusing.